doc/_templates/autoapi/module.rst

```diff
@@ -0,0 +1,108 @@
+=={{ '=' * node.name|length }}==
```

Comment:
Might be worth adding a comment at the top explaining why this custom template is required. Otherwise, we might forget the reason in the future.

Reply:
Like this:
```rst
.. # Template modified  by Patrick Lehmann
     * removed automodule on top, because private members are activated for autodoc (no doubled documentation).
     * Made sections like 'submodules' bold text, but no headlines to reduce number of ToC levels.
```